library/common/extensions/cert_validator/platform_bridge/config.cc

```diff
@@ -0,0 +1,23 @@
+#include "library/common/extensions/cert_validator/platform_bridge/config.h"
+
+#include "library/common/api/external.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace TransportSockets {
+namespace Tls {
+
+CertValidatorPtr PlatformBridgeCertValidatorFactory::createCertValidator(
+    const Envoy::Ssl::CertificateValidationContextConfig* config, SslStats& stats,
+    TimeSource& /*time_source*/) {
+  platform_validator_ =
+      static_cast<envoy_cert_validator*>(Api::External::retrieveApi("platform_cert_validator"));
```

Comment:
Considering https://github.com/envoyproxy/envoy-mobile/blob/5e8d877ae04abaa6d0d9e0f81c2d90ab456748bd/library/common/api/external.cc#L25 you may want your own guard here to ensure that the implementation is found, just for safety.